### DISCUSSION TITLE
Question: How can I disable the inline end of function or module?

### DISCUSSION THREAD

**Discussion by rluvaton** [zed-industries/zed#12328] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

I want to be able to disable the end of function text

### If applicable, add mockups / screenshots to help present your vision of the feature

![image](https://github.com/zed-industries/zed/assets/16746759/9c76a75a-50e7-4c4a-8c06-563e025c4609)

**Comment by SomeoneToIgnore** [comment]
Please consider using https://github.com/zed-industries/zed/discussions for questions.

Those are inlay hints, coming from rust-analyzer LSP server, which can be reconfigured to turn them off.
Zed has an example on how to set those in https://zed.dev/docs/languages/rust#inlay-hints and mentions the server's configuration page too: https://rust-analyzer.github.io/manual.html#inlay-hints

```
rust-analyzer.inlayHints.closingBraceHints.enable (default: true)

    Whether to show inlay hints after a closing } to indicate what item it belongs to.
```

looks like the one that's needed.
